iroha/src/smartcontracts/isi/asset.rs

```diff
@@ -159,6 +184,30 @@ pub mod isi {
         }
     }
 
+    impl<W: WorldTrait> Execute<W> for Burn<Asset, Fixed> {
+        type Error = Error;
+
+        fn execute(
+            self,
+            _authority: <Account as Identifiable>::Id,
+            wsv: &WorldStateView<W>,
+        ) -> Result<(), Error> {
+            assert_asset_type(
+                &self.destination_id.definition_id,
+                wsv,
+                AssetValueType::Fixed,
+            )?;
+            wsv.modify_asset(&self.destination_id, |asset| {
+                let quantity: &mut Fixed = asset.try_as_mut()?;
+                *quantity = quantity
+                    .checked_sub(self.object)
+                    .ok_or_else(|| error!("Not enough quantity to burn."))?;
+                Ok(())
+            })?;
+            Ok(())
```

Comment:
```suggestion
            })
```

Reply:
Changed. And all similar places in this module.